### ISSUE TITLE
No registered devices! While building provenance. 

### ISSUE THREAD

**Issue by elbarbari** [jasarien/Provenance#293]
After provenance suddenly stopped running and also Mame, I tried rebuilding it again. 

This time am getting "no registered device was found .. " or something like that. 

I have regular developers account i.e., I didn't pay $99. I just want to install provenance. 

Is apple cracking down on none subscribers?

Please help. Thank you.

**Comment by jasarien** [comment]
Is your device connected?

**Comment by elbarbari** [comment]
Yes, it is connected and sows up in devices.

**Comment by mofux** [comment]
Just had the same problem when I tried to compile for tvOS, turned out I had selected the wrong scheme. In XCode, just next to the play and stop button in the window header, there is the scheme selection dropdown. Make sure you select the appropriate scheme for your device (for me it was ProvenanceTV-Release), then the device should show up. I guess your device is not showing up because the selected scheme is not supported for your device.
